### PR TITLE
Add missing doc for get_video_backend

### PR DESCRIPTION
**Pull request by bjuncek**
addressing #3633

### CODE REVIEW

File: torchvision/__init__.py [hunk 1]
```diff
@@ -85,6 +90,12 @@ def set_video_backend(backend):
 
 
 def get_video_backend():
+    """
+    Returns the currently active video backend used to decode videos.
+
+    Returns:
+        (str): Name of the video backend. one of {'pyav', 'video_reader'}.
```
Comment (NicolasHug) [inline torchvision/__init__.py:97]:
```suggestion
        str: Name of the video backend. one of {'pyav', 'video_reader'}. 
```

**Review by NicolasHug — COMMENTED** [review]
Thanks @bjuncek , it looks like your editor automatically formatted some unrelated code: it's better to avoid this as it tends to break `git blame`. In big PRs, it can also complicate the review with noisy changes

**Comment by bjuncek** [comment]
Hi @NicolasHug - yeah makes sense. Sorry for the bother. I've reverted the change and only updated the docstring per your suggestion. 
Please let me know if it's fine now and I'll ping someone for the merge. 

Best,
Bruno

**Review by NicolasHug — APPROVED** [review]
Thanks! The docs CI was passing so I'll merge ;)

**Pull request merged**